webapp/src/main/java/com/box/l10n/mojito/service/branch/notification/slack/BranchNotificationMessageBuilderSlack.java

```diff
@@ -132,4 +135,15 @@ String getSummaryString(List<String> strings) {
         .map(s -> StringUtils.abbreviate(s, STRING_IN_SUMMARY_ABRREVIATE_LENGHT))
         .collect(Collectors.joining(", "));
   }
+
+  protected String getTranslationsReadyMsg(String branchName) {
+    GithubBranchDetails branchDetails = new GithubBranchDetails(branchName);
+    MessageFormat messageFormat = new MessageFormat(translationsReadyMsg);
+    ImmutableMap<String, Object> messageParamMap =
+        ImmutableMap.<String, Object>builder()
+            .put("branchName", branchName)
+            .put("repository", branchDetails.getRepository())
```

Comment:
can we prefix those message params with github eg. `github.repository` , `github.prNumber`, `github.owner` ?

`repository` could/would be the "mojito" repository

Reply:
I tried to rename it using github.repository but seems like `.` cannot be used there, as it caused an error. I replaced to githubRepository.